language-server/modules/langserver-core/src/test/resources/semantictokens/config/project/main.bal

```diff
@@ -1,6 +1,6 @@
 import ballerina/iox;
 import ballerina/httpx;
-import ballerina/log;
+import ballerina/logx;
```

Comment:
@IMS94 I had a chat with @malinthar  regarding this. Seems if the log module is ~/.ballerina folder, this import causes issues. So log module was renamed same as the other stdlibs module have been.